xwiki-platform-core/xwiki-platform-bridge/src/main/java/org/xwiki/bridge/event/DocumentDeletedEvent.java

```diff
@@ -42,28 +42,34 @@ public class DocumentDeletedEvent extends AbstractDocumentEvent
      * changes.
      */
     private static final long serialVersionUID = 1L;
-    
+
+    /**
+     * The reference of the deleted document.
+     */
+    private final DocumentReference sourceReference;
+
     /**
      * Constructor initializing the event filter with an
      * {@link org.xwiki.observation.event.filter.AlwaysMatchingEventFilter}, meaning that this event will match any
      * other document delete event.
      */
     public DocumentDeletedEvent()
     {
-        
+        this.sourceReference = null;
```

Comment:
This seems unnecessary

Reply:
It is not because `sourceReference` is `final` and need to be initialized explicitly.